compiler/src/org.graalvm.compiler.hotspot/src/org/graalvm/compiler/hotspot/GraalHotSpotVMConfig.java

```diff
@@ -398,14 +398,18 @@ private int getJvmciJvmtiCapabilityOffset(String name) {
     {
         int offset;
         try {
-            offset = getFieldOffset("JavaThread::_jvmci_reserved0", Integer.class, "jlong", -1, JVMCI ? jvmciGE(JVMCI_21_1_b02) : JDK >= 17);
+            offset = getFieldOffset("JavaThread::_jvmci_reserved0", Integer.class, "jlong", -1, hasJVMCIReservedFields());
```

Comment:
This depends on https://bugs.openjdk.java.net/browse/JDK-8272736 being in OpenJDK 11, which it currently isn't. Just FYI. It should do no harm, though, as it would fall back to using the `intptr_t*` type in the catch block.